bin/pyrdp-mitm.py

```diff
@@ -158,6 +159,11 @@ def main():
     parser.add_argument("-L", "--log-level", help="Console logging level. Logs saved to file are always verbose.", default="INFO", choices=["INFO", "DEBUG", "WARNING", "ERROR", "CRITICAL"])
     parser.add_argument("-F", "--log-filter", help="Only show logs from this logger name (accepts '*' wildcards)", default="")
     parser.add_argument("-s", "--sensor-id", help="Sensor ID (to differentiate multiple instances of the MITM where logs are aggregated at one place)", default="PyRDP")
+    parser.add_argument("--payload", help="Command to run automatically upon connection", default=None)
+    parser.add_argument("--payload-powershell", help="PowerShell command to run automatically upon connection", default=None)
+    parser.add_argument("--payload-powershell-file", help="PowerShell script to run automatically upon connection (as -EncodedCommand)", default=None)
+    parser.add_argument("--payload-delay", help="Time to wait after a new connection before sending the payload, in milliseconds", default=None)
```

Comment:
No point, we already do some checking. Some people might not want to run payloads.

Reply:
I meant just for the duration. If 80% of usecases are ok with say 1000ms for duration, not having to specify it would be a bit better user experience

Reply:
I think the most important part about this is safety. If duration is too short, it can reveal to the user that bad things are happening. IMO it's better to force users to think about that option than to compromise their stealthiness because of an arbitrary default value.

Reply:
Yeah makes sense said this way!